client/src/components/PlantButton/stateUtils.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    controls: {
+        water: "water",
+    },
+    plantImageUrls: [
+        "https://i.ibb.co/DVP9kkq/Tree1.png",
+        "https://i.ibb.co/CnQgYZp/Tree2.png",
+        "https://i.ibb.co/2s4TDYc/Tree3.png",
+        "https://i.ibb.co/5r5tqWY/Tree4.png",
+    ],
+    buttonImageUrls: {
+        water: "https://i.ibb.co/C04zg8V/Watering-Can.png",
+    },
+    feedback: {
+        start:
+            "Dear, When you finish your task, you will get a Water Coin, you can use this coin to plant your own trees! Try to click the button!",
+        win: "Congratulations! You are now a certified plant Mom.",
+        noCoin: "Sorry, You do not have enough coins to water the tree.",
```

Comment:
Better word: **Sorry, you need to complete more tasks to water the tree.**
I think that we should not mention the word "coin" because there is no information on the page that indicates the user has coins.